components/integrations/installed_outgoing_webhook.tsx

```diff
@@ -159,25 +177,20 @@ export default class InstalledOutgoingWebhook extends React.PureComponent<Props>
 
         let actions = null;
         if (this.props.canChange) {
-            actions = (
-                <div className='item-actions'>
+            actions =
+                (<div className='item-actions'>
                     <button
                         className='style--none color--link'
                         onClick={this.handleRegenToken}
                     >
-                        <FormattedMessage
-                            id='installed_integrations.regenToken'
-                            defaultMessage='Regen Token'
-                        />
+                        <i className='icon icon-refresh'/>
```

Comment:
please use the compass-icons components

Reply:
@WilliamLongKing the size prop should set the `width` and `height` values on the svg element, so the only reason this being ineffective is when CSS overwrites it.

For the `destructive` state you need to use the error color: `rgb(var(--error-text-color-rgb))` e.g. (you can also insert this into the `color` prop from the compass-icon component)